ebssurrogate/scripts/chroot-bootstrap.sh

```diff
@@ -0,0 +1,202 @@
+		fail2ban \
+		locales
+
+	if [ "${ARCH}" = "arm64" ]; then
+		apt-get $APT_OPTIONS --yes install linux-aws initramfs-tools dosfstools
+	fi
+}
+
+function setup_locale {
+cat << EOF > /etc/default/locale
+LANG="C.UTF-8"
+LC_CTYPE="C.UTF-8"
+EOF
+	localedef -i en_US -f UTF-8 en_US.UTF-8
+}
+
+# Disable IPV6 for ufw
+function disable_ufw_ipv6 {
+	sed -i 's/IPV6=yes/IPV6=no/g' /etc/default/ufw
+}
+
+function install_packages_for_build {
+	apt-get install -y --no-install-recommends linux-libc-dev \
+	 acl \
+	 magic-wormhole sysstat \
+	 build-essential libreadline-dev zlib1g-dev flex bison libxml2-dev libxslt-dev libssl-dev libsystemd-dev libpq-dev libxml2-utils uuid-dev xsltproc ssl-cert \
+	 llvm-11-dev clang-11 \
+	 gcc-10 g++-10 \
+	 libgeos-dev libproj-dev libgdal-dev libjson-c-dev libboost-all-dev libcgal-dev libmpfr-dev libgmp-dev cmake \
+	 libkrb5-dev \
+	 maven default-jre default-jdk \
+	 curl gpp apt-transport-https cmake libc++-dev libc++abi-dev libc++1 libglib2.0-dev libtinfo5 libc++abi1 ninja-build python \
+	 liblzo2-dev
+}
+
+function setup_grub_conf_arm64 {
+cat << EOF > /etc/default/grub
+GRUB_DEFAULT=0
+GRUB_TIMEOUT=0
+GRUB_TIMEOUT_STYLE="hidden"
+GRUB_DISTRIBUTOR="Supabase postgresql"
+GRUB_CMDLINE_LINUX_DEFAULT="nomodeset console=tty1 console=ttyS0"
+EOF
+}
+
+function setup_grub_conf_amd64 {
+	mkdir -p /etc/default/grub.d
+
+cat << EOF > /etc/default/grub.d/50-aws-settings.cfg
+GRUB_RECORDFAIL_TIMEOUT=0
+GRUB_TIMEOUT=0
+GRUB_CMDLINE_LINUX_DEFAULT=" root=/dev/nvme0n1p2 rootfstype=ext4 rw noatime,nodiratime,discard console=tty1 console=ttyS0 ip=dhcp tsc=reliable net.ifnames=0 quiet module_blacklist=psmouse,input_leds,autofs4 ipv6.disable=1 nvme_core.io_timeout=4294967295 systemd.hostname=ubuntu"
+GRUB_TERMINAL=console
+GRUB_DISABLE_LINUX_UUID=true
+EOF
+	# remove initd
+	#rm -f /boot/microcode.cpio
+	#rm -rf /boot/initrd.*
+}
+
+# Install GRUB
+function install_configure_grub {
+	if [ "${ARCH}" = "arm64" ]; then
+		apt-get $APT_OPTIONS --yes install cloud-guest-utils fdisk grub-efi-arm64
+		setup_grub_conf_arm64
+		rm -rf /etc/grub.d/30_os-prober
+		sleep 1
+	fi
+	grub-install /dev/xvdf && update-grub
+}
+
+# skip fsck for first boot
+function disable_fsck {
+	touch /fastboot
+}
+
+# Don't request hostname during boot but set hostname
+function setup_hostname {
+	sed -i 's/gethostname()/ubuntu /g' /etc/dhcp/dhclient.conf
+	sed -i 's/host-name,//g' /etc/dhcp/dhclient.conf
+	echo "ubuntu" > /etc/hostname
+	chmod 644 /etc/hostname
+}
+
+# Set options for the default interface
+function setup_eth0_interface {
+cat << EOF > /etc/netplan/eth0.yaml
+network:
+  version: 2
+  ethernets:
+    eth0:
+      dhcp4: true
+EOF
+}
+
+function disable_sshd_passwd_auth {
+	sed -i -E -e 's/^#?\s*PasswordAuthentication\s+(yes|no)\s*$/PasswordAuthentication no/g' \
+	  -e 's/^#?\s*ChallengeResponseAuthentication\s+(yes|no)\s*$/ChallengeResponseAuthentication no/g' \
+	 /etc/ssh/sshd_config
+}
+
+function create_admin_account {
+	groupadd admin
```

Comment:
what's this group used for?

Reply:
Without this `admin` group `tasks/internal/admin-api.yml` fails with error like "Group admin does not exist"